docs/source/examples/notebooks/getting_started/tutorial-4-setting-parameter-values.ipynb

```diff
@@ -278,54 +308,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "More details on each subset can be found [here](https://github.com/pybamm-team/PyBaMM/tree/develop/pybamm/input/parameters)."
-   ]
-  },
-  {
-   "attachments": {},
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Change individual parameters"
-   ]
-  },
-  {
-   "attachments": {},
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "We often want to quickly change a small number of parameter values to investigate how the behaviour or the battery changes. In such cases, we can change parameter values without having to leave the notebook or script you are working in. \n",
+    "## Change individual parameters\n",
```

Comment:
should we show `print_parameter_info` here to show which parameters can and cannot be functions?